osu.Game/Graphics/UserInterface/OsuAnimatedButton.cs

```diff
@@ -111,6 +111,10 @@ protected override void OnHoverLost(HoverLostEvent e)
 
         protected override bool OnClick(ClickEvent e)
         {
+            // Handle case where a click is triggered via TriggerClick().
+            if (!IsHovered)
+                hover.FadeOutFromOne(1600);
```

Comment:
Is this supposed to have no easing?

Reply:
Yeah, just like the `Flash` it's trying to show doesn't.

Unfortunately in the case of this PR, it doesn't work great because at the same time the button is getting regenerated to show a different icon / colour state.